src/sharing-service.ts

```diff
@@ -327,14 +343,21 @@ export class SharingService {
     const endpoint = new URL(`notebooks/${id}`, this.api_url);
 
     const token = await this.token;
+    const body = JSON.stringify(requestData);
     const response = await fetch(endpoint, {
       method: 'PUT',
       headers: await this.makeHeaders(token),
-      body: JSON.stringify(requestData)
+      body
     });
 
     if (!response.ok) {
-      throw new Error(`Updating notebook failed: ${response.statusText}`);
+      const sizeInMB = new Blob([body]).size / (1000 * 1000);
```

Comment:
I thought it was MiB, too. Thanks for the catch. I think we should report it upstream (or fix it, given that something like this should be trivial).